media-sound/picard/picard-1.4.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+DISTUTILS_SINGLE_IMPL=1
+DISABLE_AUTOFORMATTING=true
+inherit eutils distutils-r1 readme.gentoo-r1
+
+DESCRIPTION="A cross-platform music tagger"
+HOMEPAGE="https://picard.musicbrainz.org"
+SRC_URI="https://github.com/metabrainz/${PN}/archive/release-${PV}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="+acoustid +cdda nls"
+
+DEPEND="dev-python/PyQt4[X,${PYTHON_USEDEP}]
+	dev-qt/qtgui:4[accessibility]
+	media-libs/mutagen
+	acoustid? ( >=media-libs/chromaprint-1.0[tools] )
+	cdda? ( dev-python/python-discid )"
+RDEPEND="${DEPEND}"
+
+RESTRICT="test" # doesn't work with ebuilds
+S=${WORKDIR}/${PN}-release-${PV}
+DOCS="AUTHORS.txt NEWS.txt"
+
+DOC_CONTENTS="If you are upgrading Picard and it does not start,
+try removing Picard's settings:
+    rm ~/.config/MusicBrainz/Picard.conf
+
+You should set the environment variable BROWSER to something like
+    firefox '%s' &
+to let python know which browser to use."
```

Comment:
Upstream at https://picard.musicbrainz.org/docs/faq/#which-browser states that the default browser can be determined automatically on Windows, GNOME and KDE systems. What about other desktop environments? I'll push a new commit removing this documentation however.